contrib/net/http/option.go

```diff
@@ -51,6 +52,7 @@ func defaults(cfg *config) {
 		cfg.analyticsRate = globalconfig.AnalyticsRate()
 	}
 	cfg.serviceName = namingschema.ServiceName(defaultServiceName)
+	cfg.isStatusError = func(status int) bool { return status >= 500 && status < 600 }
```

Comment:
This is fine as-is, but to conform with other contribs and make it cleaner, you can consider defining the functionality in a separate function called `isServerError`, see other examples here: https://github.com/DataDog/dd-trace-go/blob/v1.70.1/contrib/gofiber/fiber.v2/option.go#L37